core/build/webpack.base.config.ts

```diff
@@ -117,7 +117,8 @@ export default {
       'theme/resource': themeResources,
 
       // Backward compatible
-      '@vue-storefront/core/store/lib/multistore': path.resolve(__dirname, '../lib/multistore.ts')
+      '@vue-storefront/core/store/lib/multistore': path.resolve(__dirname, '../lib/multistore.ts'),
+      'src/modules/order-history/components/UserOrders': path.resolve(__dirname, '../../core/modules/order/components/UserOrdersHistory')
```

Comment:
@filrak  is it correct? The name of the file has changed as well as the name of exported object.

Reply:
yup ;)